controllers/permits/special-uses/validate.js

```diff
@@ -36,36 +36,143 @@ const phoneNumber = schema.phoneNumber;
 
 //*******************************************************************
 
-function removeInstance(prop){
+function zipFormat(input){
+	
+	let valid = true;
 
-	let fixedProp = '';
-	if (prop.indexOf('.') !== -1){
+	if (typeof input === 'number'){
 
-		fixedProp = prop.substring((prop.indexOf('.') + 1), (prop.length));
+		const inputStr = input + '';
+
+		if (!inputStr.match(/^[0-9]{5}$|^[0-9]{9}$/)){
+
+			valid = false;
+
+		}
 
 	}
+	return valid;
+}
+function areaCodeFormat(input){
 
-	return fixedProp;
+	let valid = true;
+
+	if (typeof input === 'number'){
+
+		const inputStr = input + '';
+		if (!inputStr.match(/^[0-9]{3}$/)){
+
+			valid = false;
+
+		}
+
+	}
+
+	return valid;
 
 }
+function phoneNumberFormat(input){
+
+	let valid = true;
 
-function getRoute(req){
+	if (typeof input === 'number'){
 
-	const path = req.originalUrl;
-	const parts = path.split('/');
-	let route;
-	if (path.charAt(path.length - 1) === '/'){
+		const inputStr = input + '';
+		if (!inputStr.match(/^[0-9]{7}$/)){
 
-		route = parts[parts.length - 2];
+			valid = false;
+
+		}
 
 	}
-	else {
 
-		route = parts[parts.length - 1];
+	return valid;
+
+}
+
+function stateFormat(input){
```

Comment:
Should there be a better state abbreviation check with the states list in a JSON?